src/dkg_hybriddkg.erl

```diff
@@ -349,79 +363,79 @@ verification_key(OutputCommitment) ->
     dkg_commitmentmatrix:lookup([1, 1], dkg_commitment:matrix(OutputCommitment)).
 
 -spec shard(#dkg{}) -> erlang_pbc:element().
-shard(_DKG=#dkg{vss_results=R, u=U}) ->
+shard(_DKG=#dkg{shares_results=R, u=U}) ->
     Zero = erlang_pbc:element_set(erlang_pbc:element_new('Zr', U), 0),
     maps:fold(fun(_K, {_, Si}, Acc) ->
                         erlang_pbc:element_add(Acc, Si)
               end, Zero, R).
 
--spec l_next(pos_integer(), pos_integer()) -> pos_integer().
-l_next(L, N) ->
+-spec leader_cap(pos_integer(), pos_integer()) -> pos_integer().
+leader_cap(L, N) ->
     case L - 1 < 1 of
         true ->
             N;
         false ->
             L - 1
     end.
 
--spec count_echo(dkg(), qset()) -> non_neg_integer().
-count_echo(_DKG=#dkg{elq=Elq, leader=Leader}, Q0) ->
+-spec count_echo(dkg(), node_set()) -> non_neg_integer().
+count_echo(_DKG=#dkg{echo_counts=EchoCount, leader=Leader}, Q0) ->
     Q = lists:usort(Q0),
-    length(maps:get({Leader, Q}, Elq, [])).
+    length(maps:get({Leader, Q}, EchoCount, [])).
 
--spec get_echo(dkg(), qset()) -> [echo()].
-get_echo(_DKG=#dkg{elq=Elq, leader=Leader}, Q0) ->
+-spec get_echo(dkg(), node_set()) -> [echo()].
+get_echo(_DKG=#dkg{echo_counts=EchoCount, leader=Leader}, Q0) ->
     Q = lists:usort(Q0),
-    maps:get({Leader, Q}, Elq, []).
+    maps:get({Leader, Q}, EchoCount, []).
 
--spec update_elq(dkg(), pos_integer(), signed_echo()) -> {true, dkg()} | false.
-update_elq(DKG=#dkg{elq=Elq}, Sender, {signed_echo, Q0}=EchoMsg) ->
+-spec update_echo_counts(dkg(), pos_integer(), signed_echo()) -> {true, dkg()} | false.
+update_echo_counts(DKG=#dkg{echo_counts=EchoCount}, Sender, {signed_echo, Q0}=EchoMsg) ->
```

Comment:
I see a few more function heads (and some map calls) have `Q` or `Q0` lingering, depending on the context, I'd imagine it's either an Echo or a Ready..